test/param/loop/paramForLoopWrongTypeLow.good

```diff
@@ -0,0 +1 @@
+paramForLoopWrongTypeLow.chpl:1: error: Invalid use of count range in param for loop
```

Comment:
I think that cases like this should actually be generating a message more like "Param loop range bounds must be integers of compatible types" or perhaps "Range bounds must be integers of compatible types in param for loop
" for compatibility with the other cases we've seen.  Specifically, the source of the error in the user's code is nothing specific to the `#2` (the count operator), but rather with the original range itself `""..4#2` to which the count operator is being applied.